README.mkd

```diff
@@ -291,6 +291,20 @@ Vendors should also aim to follow the conventions used for naming mnemonics
 in the ratified base ISA and extensions (e.g. the use of 'w', 'd',
 'u', and 's' suffixes).
 
+### CSR naming scheme
+
+Vendors may define their own CSRs within the custom read-only CSR address range
+specified in the RISC-V ISA spec. However, to avoid conflicts, each vendor CSR
+must include a prefix corresponding to the vendor's name.
+
+The vendor prefix should match the prefix defined in assembly mnemonics. The
```

Comment:
Looks good to me. Are there any rules for CSR addressing?

Reply:
ISA spec has define custom/vendor CSR addressing range[1], but I don't really care the CSR is defined within the range or not...since that's vendor's issue rather than open source toolchain maintainer's issue.

[1] https://github.com/riscv/riscv-isa-manual/blob/main/src/priv-csrs.adoc#csr-listing

Reply:
Thanks for updating the specs, @kito-cheng!